ui/src/components/review-board-header.tsx

```diff
@@ -0,0 +1,34 @@
+'use client';
+
+import Text from '@/components/atomic/text';
+import { BoardHeader } from '@/components/board-header';
+import SearchForm from '@/components/search-form';
+import Link from 'next/link';
+import { useState } from 'react';
+
+export function ReviewBoardHeader() {
+  const [isSearching, setIsSearching] = useState(false);
+
+  return (
+    <BoardHeader>
+      <div className="flex w-full flex-col">
+        <div className="flex justify-between">
+          <button
+            className="h-fit w-fit rounded-full border bg-white px-2 py-1"
+            onClick={() => (isSearching ? setIsSearching(false) : setIsSearching(true))}
```

Comment:
조금 더 간단하게 적을 수 있을 것 같아 제안드려봅니다.
```suggestion
            onClick={() => setIsSearching(!isSearching)}
```

Reply:
그렇네요! 09632b8 에서 반영했습니다. 감사합니다.